utils/ah_lut.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+(c) Copyright 2018 Sensirion AG, Switzerland
```

Comment:
We have 2020 now :slightly_smiling_face: Also should we include the full license header as in the c files?

Reply:
Yes, full license makes sense. On the year, I'm not a fan of updating copyrights without actual changes.. as in, I'm all in favor of actually having the copyright expire eventually
https://en.wikipedia.org/wiki/List_of_countries%27_copyright_lengths

Reply:
I also don't care that much about the year. Feel free to add full license header or just resolve.